test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/main.py

```diff
@@ -905,6 +887,27 @@ def add_option_alias_name(name, option_name):
                 msg=msg,
             )
 
+    def _validate_return_docs(self, returns: t.Any, context: t.Optional[list[str]] = None) -> None:
```

Comment:
Use `object` insstead of `t.Any` when practical. In this case, since `isinstance` is being used to narrow the type, it works without further code changes. Also, use the newer `|` syntax for optional types.

```suggestion
    def _validate_return_docs(self, returns: object, context: list[str] | None = None) -> None:
```

Reply:
I guess `|` is allowed here since this is "controller only" code?

Reply:
It's supported by all Python versions supported in devel, not just controller-only code.